service/test/integration2_test.go

```diff
@@ -1904,3 +1903,26 @@ func TestStopRecording(t *testing.T) {
 		assertNoError(err, t, "GetState()")
 	})
 }
+
+func TestIssue1955(t *testing.T) {
```

Comment:
I'd like to break the pattern of naming tests after Github issues. Let's just call it `TestClearLogicalBreakpoint`.